cantabular/table.go

```diff
@@ -298,11 +298,12 @@ func decodeValues(ctx context.Context, dec jsonstream.Decoder, dims Dimensions,
 // createCSVHeader creates an array of strings corresponding to a csv header
 // where each column contains the value of the corresponding dimension, with the last column being the 'count'
 func createCSVHeader(dims Dimensions) []string {
-	header := make([]string, len(dims)+1)
+	l := len(dims)
+	header := make([]string, l+1)
 	for i, dim := range dims {
-		header[i+1] = dim.Variable.Label
+		header[i] = dim.Variable.Label
 	}
-	header[0] = "count"
+	header[l] = "Observation"
```

Comment:
Query: Should this be "Observation" or "Observations"
Ticket says "Observation" but I'd just like it checked

Reply:
All good - ignore me